Assets/Scripts/BossBullet.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BossBullet : MonoBehaviour
+{
+    public float speed = 10f;
+    public GameObject main;
```

Comment:
In C# `Main` ist der Methodenname für die `static` Start Funktion, bitte Variable umbennen.